bodhi/server/models.py

```diff
@@ -2323,11 +2323,14 @@ def check_karma_thresholds(self, db, agent):
                     "%s update has reached the stable karma threshold and can be pushed to "
                     "stable now if the maintainer wishes") % self.title)
         elif self.unstable_karma and self.karma <= self.unstable_karma:
-            log.info("Automatically unpushing %s" % self.title)
-            self.obsolete(db)
-            notifications.publish(
-                topic='update.karma.threshold.reach',
-                msg=dict(update=self, status='unstable'))
+            if self.status is UpdateStatus.pending and not self.autokarma:
+                pass
```

Comment:
For pending updates with unstable karma (but with disabled autopush), we would not like to send it to `obsolete` state automatically https://github.com/fedora-infra/bodhi/blob/develop/bodhi/tests/server/services/test_updates.py#L2423